src/http/Message.h

```diff
@@ -66,6 +66,11 @@ class Message : public RefCountable
      */
     bool persistent() const;
 
+    /// Decrements the message lock count, deletes the message
+    /// when the lock count is zero.
+    /// \returns true if the message was deleted (or is nil)
+    static inline bool Destroy(Message *);
```

Comment:
C++ faq [recommends](https://isocpp.org/wiki/faq/inline-functions#where-to-put-inline-keyword) avoid putting 'inline' here.

Reply:
I agree with that FAQ entry rationale, but is there a C++ standard rule that allows us to omit `inline` here without affecting the compiled code? In other words, what C++ standard rule/requirement allows us to omit `inline` in the method declaration while keeping the effects of that `inline` intact?

Reply:
This rule is described in section [10.3.1.2](http://eel.is/c++draft/class.mfct#2) of the standard (I found its draft, though).

Reply:
Excellent. Please remove the `inline` I added.